app/models/user.rb

```diff
@@ -18,6 +18,12 @@ class User < ActiveRecord::Base
   validates :email, uniqueness: true, presence: true, on: :create, email: true
   validates :email, email: true, on: :update
 
+  scope :for_review_counters, -> { joins(:cards).
```

Comment:
Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.